articles/libraries/auth0-android/index.md

```diff
@@ -282,6 +282,10 @@ To ensure an Open ID Connect compliant response you must either request an `audi
 
 ### Login with database connection
 
+::: panel-warning Database authentication on Native Platforms
```

Comment:
Is there a reason you went with a `panel` instead of a `warning` element for this one since you did with the others below? You're probably at about the size of warning where you could use either, but I'd use one consistently since it's the same message.

Reply:
I did it for consistency, because some lines below "### Passwordless login" section has the same panel-warning for a similar message.

Reply:
Fair enough. Typically the rule of thumb is `warning` or `note` are for short-form content like 4 lines or less, `panel-foo` are for longer form, or for things broken up into more than one paragraph.

In any case, this seems fine for now, was just checking.